ooniprobe/Utility/UIDevice-DeviceInfo.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
```

Comment:
I'd like to learn something today. Why is the file named `UIDevice-DeviceInfo`? Is this because UIDevice implements the DeviceInfo protocol? (Does this make sense?)

Reply:
Because it's a subclass of UIDevice and @hellais didn't want to call it UIDevice-Hardware as the library and he proposed DeviceInfo :D

Reply:
Ok thanks for the context. I agree to name it differently FWIW.